app/src/main/java/com/ncnf/event/EventActivity.java

```diff
@@ -23,30 +23,30 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_event);
 
         String event_uid = getIntent().getStringExtra(UUID_KEY);
-        Event event = db.getEvent(event_uid);
-        if (event == null) {
+        Social social = db.getEvent(event_uid);
```

Comment:
Will the same view be used for Event and Group ? In that case we could rename the activity to SocialActivity.

Reply:
Fixed !